### PR TITLE
[tests] Fix pylint errors in `tests/interactive`

### PR DESCRIPTION
**Pull request by cidrblock**
Related #643 

- does what it says
- formatted/added specifics to doc strings as encountered
- remove unneeded pylint disables as encountered
- one constant name correction made when seen

### CODE REVIEW

File: tests/integration/_action_run_test.py [hunk 1]
```diff
@@ -137,23 +140,26 @@ def run_action_stdout(self, **kwargs) -> Tuple[int, str, str]:
         sys.stdin = stty  # type: ignore
 
         # set stderr and stdout to fds
-        sys.stdout = tempfile.TemporaryFile()  # type: ignore
-        sys.stderr = tempfile.TemporaryFile()  # type: ignore
+        with tempfile.TemporaryFile() as sys_stdout:
```
Comment (webknjaz) [inline tests/integration/_action_run_test.py:143]: I think you could merge two with-blocks into one.

File: tests/integration/_tmux_session.py [hunk 2]
```diff
@@ -161,10 +164,14 @@ def interaction(
         ignore_within_response=None,
         timeout=300,
     ):
-        """interact with the tmux session
+        # pylint: disable=too-many-branches
```
Comment (webknjaz) [inline tests/integration/_tmux_session.py:167]: Is it a conscious choice not to have coma-separated disables?

Comment (cidrblock) [reply tests/integration/_tmux_session.py:167]: yeah, I prefer one liners

File: tests/integration/_action_run_test.py (hunk 1 above)
Comment (cidrblock) [reply tests/integration/_action_run_test.py]: good idea

**Pull request merged**